scrunch/datasets.py

```diff
@@ -1879,7 +1879,10 @@ def __iter__(self):
         """
         Iterable of subvariable aliases
         """
-        for svar in self.resource.subvariables.index.values():
+        values = self.resource.subvariables.index.values()
+        sort_func = lambda x: x['id']
+
+        for svar in sorted(values, key=sort_func):
```

Comment:
Now using the order from the elements in the subvariables member of the variable entity.